### ISSUE TITLE
Request : PrivateBox

### ISSUE THREAD

**Issue by homersimpsons** [qklabs/qksms#377]
Hello, Thank you for QK,
Can you, please, add a Private Message Box, accessible only with a Schema or Pin code. Contacts in this need to be specified, when selected it would automatically add all te message in the Private Box. Furhermore user can pick out a contact from private box to restore their conversation in Normal Box.

**Comment by homersimpsons** [comment]
Maybe it could be easier to protect the entire app instead of just some conversation to begin

**Comment by dejageek** [comment]
I agree, this feature would be nice.  It could act much like the blocked feature, but more "stealthy".  Customizable notifications would be a nice addition to it

**Comment by NovaViper** [comment]
Any progress on this issue?
